test/testkit/MockitoSugar.scala

```diff
@@ -23,8 +23,8 @@ import scala.concurrent.Future
 
 trait MockitoSugar extends org.scalatest.mock.MockitoSugar {
   // Unfortunately we cannot use the `when().thenReturn` format for spies.
-  def doReturnAsync() = Mockito.doReturn(Future.successful(()))
-  def doReturnAsync[A](value: A) = Mockito.doReturn(Future.successful(value))
+  def doReturnAsync() = Mockito.doReturn(Future.successful(()), Nil.toSeq:_*)
```

Comment:
This is the only Mockito 2 strangeness i encountered. doReturn now has a second, varArgs* argument, and the compiler had conflicts when only one argument was specified. I've specified an empty varArgs set as a result, but i'd be open to anyone who knows a more concise way of doing that than Nil.toSeq:_*